package.json

```diff
@@ -50,6 +50,7 @@
     "semantic-release": "semantic-release"
   },
   "dependencies": {
+    "@kong-ui-public/analytics-utilities": "^0.7.2",
```

Comment:
Kongponents cannot have a dependency on this package - this repo is completely isolated from other kong component libraries

Reply:
Understood